lib/replace_parameters.js

```diff
@@ -506,6 +496,8 @@ module.exports = class ParameterReplacer {
                 }
                 if (slot.type.isLocation && !this._replaceLocations)
                     continue;
+                if ( (slot.type.isNumber || slot.type.type==='tt:phone_number') && !this._replaceNumbers)
+                    continue;
```

Comment:
Phone numbers are not numbers, they are PHONE_NUMBER_ tokens and are always preprocessed by the tokenizer.

If you want to handle PHONE_NUMBER_, you'll need to handle EMAIL_ADDRESS_, URL_ & PATH_NAME_ too.

Reply:
I am only doing so since numbers are problematic in translation.
As I mentioned above I only replace numbers before translation and requoted them after translation so other scripts will always see placeholders.

Reply:
Yes I'm saying (here and the comment about lib/dataset_augmenter.js). you'll have problems in this script. Problems you don't see cause you have a limited dataset (restaurants) but will show up once you use it on a larger dataset (like Thingpedia).

Reply:
Gotcha, the only answer I have now is replace_numbers only guaranteed to work for restaurant datasets :)) Can put a warning that "Be advised that it might not work on datasets containing measurements, etc."

Reply:
You can guess my answer ;)